src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -1062,22 +1093,22 @@ $dark-focusColor: var(--green-500, color(green-500));
 
 .k-code-block-secondary-actions {
   display: flex;
-  gap: var(--spacing-xxs, spacing(xxs));
+  gap: var(--spacing-xxs, var(--kui-space-20, $kui-space-20));
   position: absolute;
-  right: var(--spacing-md, spacing(md));
-  top: var(--spacing-xs, spacing(xs));
+  right: var(--spacing-md, 16px);
+  top: var(--spacing-xs, 8px);
```

Comment:
**Suggestion**: Should this use the same fallback style instead of a fixed pixel value?

```suggestion
  top: var(--spacing-xs, var(--kui-space-40, $kui-space-40));
```

Reply:
See comment above